microcosm_pubsub/dispatcher.py

```diff
@@ -62,7 +62,7 @@ def handle_batch(self, bound_handlers) -> List[MessageHandlingResult]:
                 message_batch_size=message_batch_size,
                 batch_elapsed_time=batch_elapsed_time,
             )
-            self.logger.info(message)
+            self.logger.debug(message)
```

Comment:
@GabeLambda  These are the batch logs you added to get stats on sqs batch publishing times. A) I think we can ultimately catch this in jaeger B) do you really need this log right now or do you feel like you've done enough tuning with the batch sizes for now?

Reply:
They're debug. I'm getting additional logs for them in prometheus.